include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -48,7 +48,9 @@ using __has_known_identity =
     typename ::std::conjunction<
         ::std::is_arithmetic<_Tp>, __dpl_sycl::__has_known_identity<_BinaryOp, _Tp>,
         ::std::disjunction<::std::is_same<typename ::std::decay<_BinaryOp>::type, ::std::plus<_Tp>>,
+                           ::std::is_same<typename ::std::decay<_BinaryOp>::type, ::std::plus<void>>,
```

Comment:
Should these additional cases be added to the #else branch below?

Reply:
Agreed, fixed.